GitFyle.Core.Api/Models/Foundations/Configurations/Exceptions/ConfigurationDependencyException.cs

```diff
@@ -0,0 +1,15 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using Xeptions;
+
+namespace GitFyle.Core.Api.Models.Foundations.Configurations.Exceptions
+{
+    public class ConfigurationDependencyException : Xeption
+    {
+        public ConfigurationDependencyException(string message, Xeption innerException) :
+            base(message, innerException)
+        { }
```

Comment:
```suggestion
        public ConfigurationDependencyException(string message, Xeption innerException)
            : base(message, innerException)
        { }
```

Reply:
@pratikkaje This and the following comments are made because each of the exception models were inconsistant to each other in formatting. Not that one is wrong or right.
My suggested is what is used across all the Standard repositories.

Reply:
Updated formatting as per your suggestion.